src/Refitter.Core/Refitter.Core.csproj

```diff
@@ -23,4 +23,8 @@
     </AssemblyAttribute>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
@ryanheath We must include the liquid templates as embedded resources in the source generator project `src/Refitter.SourceGenerator/Refitter.SourceGenerator.csproj` as well

Reply:
Hmm, is Refitter.SourceGenerator.csproj not referencing Refitter.Core.csproj?
The way the resources are loaded, is that they are bound to assemblies names.

Reply:
@ryanheath No, files are included as links instead. It was a bit tricky to get the source generator up and running and we kind of got stuck where we don't add files directly to the source generator project but only work on the core project. 

I should probably re-visit the source generator project setup again. You can skip the change request, I'll take care of it once we merge this in

Reply:
I have made the change request, it was needed anyway to get the source generator work the same as core.